app/common/state/tabContentState.js

```diff
@@ -152,7 +152,8 @@ const tabContentState = {
       // If closeIcon is fixed then there's no room for another icon
       !tabContentState.hasFixedCloseIcon(state, frameKey) &&
       // completely hide it for small sizes
-      !hasBreakpoint(frame.get('breakpoint'), ['mediumSmall', 'small', 'extraSmall', 'smallest'])
+      !hasBreakpoint(frame.get('breakpoint'),
+        ['medium', 'mediumSmall', 'small', 'extraSmall', 'smallest'])
```

Comment:
I checkout my PR that refactored this function and I see that `medium` was not there before either. How come that we need to add it now? Was this working correctly before in previous versions or not?